ocaml/file_formats/cmt_format.ml

```diff
@@ -178,15 +175,15 @@ let save_cmt filename modname binary_annots sourcefile initial_env cmi shape =
            | Some cmi -> Some (output_cmi temp_file_name oc cmi)
          in
          let source_digest = Option.map Digest.file sourcefile in
+         let compare_imports import1 import2 =
+           let modname1 = Import_info.name import1 in
+           let modname2 = Import_info.name import2 in
+           Compilation_unit.Name.compare modname1 modname2
+         in
          let get_imports () =
            Env.imports ()
-           |> List.map (fun import ->
-                let name = Import_info.name import in
-                let crc_with_unit = Import_info.crc_with_unit import in
-                name, crc_with_unit)
-         in
-         let compare_imports (modname1, _crc1) (modname2, _crc2) =
-           Compilation_unit.Name.compare modname1 modname2
+           |> List.sort compare_imports
```

Comment:
(If we expect long list, it might
be beneficial to convert to an
array before sorting.)

Reply:
Fixed